variables.tf

```diff
@@ -44,6 +44,27 @@ variable "additional_public_subnets" {
   }))
 }
 
+variable "bastion_host_ami_filter" {
+  description = "List of maps used to create the AMI filter for the action runner AMI."
+  type        = map(list(string))
+
+  default = {
+    name = ["amzn2-ami-hvm-2.*-x86_64-ebs"]
+  }
+}
+
+variable "bastion_host_ami_owners" {
+  description = "The list of owners used to select the AMI of action runner instances."
+  type        = list(string)
+  default     = ["amazon"]
+}
+
+variable "bastion_host_ami_id" {
+  type        = string
+  description = "AMI to use for the instance. Setting this will ignore `ami_filter` and `ami_owners`."
```

Comment:
Naming may need updates (i.e. `ami_name_filter`).

Reply:
Also I'd prefer something like "The ID of the AIM to use for the instance. (...)".